### PR TITLE
fix spelling in module docs

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

**Review by schurzi — COMMENTED** [review]
inconsistent use of `-` as I understand they should be everywhere, when there are multiple lines/sentences?

**Comment by rndmh3ro** [comment]
> inconsistent use of `-` as I understand they should be everywhere, when there are multiple lines/sentences?

I fixed hopefully all inconsistences. Can you please re-check?

**Review by schurzi — APPROVED** [review]

File: plugins/modules/icinga_command_template.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 DOCUMENTATION = """
 ---
 module: icinga_command_template
-short_description: Manage command templates in Icinga2
+short_description: Manage command templates in Icinga2.
```
Comment (felixfontein) [inline plugins/modules/icinga_command_template.py:33]: The short description is the only thing that should not end in a period.

Comment (rndmh3ro) [reply plugins/modules/icinga_command_template.py:33]: Thanks, I'll fix that.

Comment (dmsimard) [reply plugins/modules/icinga_command_template.py:33]: @felixfontein is that something that you have to know or is it picked up by linting/sanity/etc ?

Comment (felixfontein) [reply plugins/modules/icinga_command_template.py:33]: @dmsimard that's something you have to know, or use @Andersson007's checking tool for.

Comment (rndmh3ro) [reply plugins/modules/icinga_command_template.py:33]: Well, TIL: https://github.com/Andersson007/ansible_reviewer

Is this tool something I can rely on that it finds things that are worth changing?

Comment (dmsimard) [reply plugins/modules/icinga_command_template.py:33]: @rndmh3ro to my knowledge the tool is not used in an official capacity right now but we are, generally speaking, looking at opportunities to automate these kind of checks in order to make it easier for collection maintainers to test and monitor for policy and best practices compliance.

**Pull request merged**